arena.js

```diff
@@ -0,0 +1,39 @@
+function toTitleCase(str)
+{
+    return str.replace(/\w\S*/g, function(txt){return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();});
```

Comment:
this looks a little complicated, i'd recommend just a function that returns `string[0].toUpperCase() + string.slice(1)`